### ISSUE TITLE
continuousnumberingwithcolumns doesn't work if numbering starts with a single-column text?

### ISSUE THREAD

**Issue by ndlecic** [maieul/ledmac#665]
Hi Maïeul,

Sorry if I'm missing something obvious, but I'd say that `continuousnumberingwithcolumns` doesn't work if a numbering sequence is started with a single-column text. Please take a look at this example, derived from `4-reledpar_column_mix_with_not_column-continuous-numbering.tex`:

```latex
\documentclass{article}
\usepackage[auxdir=aux,noend,nofamiliar,series={A},noeledsec]{reledmac}
\usepackage{polyglossia}
\setmainlanguage{latin}
\setotherlanguage{english}
\usepackage{libertineotf}
\usepackage[widthliketwocolumns,continuousnumberingwithcolumns]{reledpar}
\setlength{\Lcolwidth}{0.47\textwidth}
\setlength{\Rcolwidth}{0.47\textwidth}
\columnsposition{c}
\setlength{\beforecolumnseparator}{0.03\textwidth}
\setlength{\aftercolumnseparator}{0.03\textwidth}
\AtBeginPairs{\sloppy}
\usepackage{lipsum}
\firstlinenum*{1}
\linenumincrement*{1}

\begin{document}

\beginnumbering
\pstart
    Ipsum sit amet lorem.
\pend
\pausenumbering

\begin{pairs}  
\begin{Leftside} 
    \resumenumbering  
    \pstart  
        Lorem ipsum.
    \pend  
    \pausenumbering
\end{Leftside} 
\begin{Rightside}   
    \resumenumbering  
    \pstart  
        Lorem ipsum dolor sit amet.
    \pend  
    \pausenumbering
\end{Rightside}  
\end{pairs}  
\Columns

\resumenumbering
\pstart  
    Lorem \edtext{ipsum}{\Afootnote{ipsam}} dolor.
\pend  
\pausenumbering

\end{document}
```
It leads to this error:

    ! Undefined control sequence.
    \resumenumbering ...rr@numberingShouldHaveStarted
                                                      \endnumberingR \beginnumbe...
    l.36 \resumenumbering

**Comment by maieul** [comment]
For the right column, you must use
\beginnumbering at the first step.

In monocolumn mode, the text is considered to be in the left colum. So in you pairs, left column have started, but not the right.

But there were an error in the message of error!

cf aa8c6969073ea6e46c5848bd2ef00eb158322889

**Comment by maieul** [comment]
aa8c6969073ea6e46c5848bd2ef00eb158322889

**Comment by ndlecic** [comment]
Oh, it didn't even cross my mind to think that way. :) With aa8c696 it works.

> For the right column, you must use
> \beginnumbering at the first step.
> In monocolumn mode, the text is considered to be in the left colum. 
> So in you pairs, left column have started, but not the right.

IMHO it would be very useful if you could add this explanation to the documentation.

Thanks!

**Comment by maieul** [comment]
could you write the handbook? I will add it.

**Comment by ndlecic** [comment]
> could you write the handbook? I will add it.

Sure. I'm trying to add a new example to `4-reledpar_column_mix_with_not_column-continuous-numbering.tex` to demonstrate the use of `\beginnumbering` in the right column, however I encountered a new problem which need your clarification. 

If I stop numbering and then start it again, I get very weird lineation. I guess one shouldn't use `\endnumbering` with `continuousnumberingwithcolumns` at all, but, then, what's the recommended way to reset lineation? Please take a look at the new section in this example:
```latex
\documentclass{article}
\usepackage[noend,nofamiliar,series={A},noeledsec]{reledmac}
\usepackage{polyglossia}
\setmainlanguage{latin}
\setotherlanguage{english}
\usepackage{libertineotf}
\usepackage[widthliketwocolumns,continuousnumberingwithcolumns]{reledpar}
\setlength{\Lcolwidth}{0.47\textwidth}
\setlength{\Rcolwidth}{0.47\textwidth}
\columnsposition{c}
\setlength{\beforecolumnseparator}{0.03\textwidth}
\setlength{\aftercolumnseparator}{0.03\textwidth}

\AtBeginPairs{\sloppy}
\usepackage{lipsum}
\firstlinenum*{1}
\linenumincrement*{1}
\begin{document}

\begin{english}
\date{}
\title{Mixing multiple columns with a single column, using continuous numbering}
\begin{abstract}
In this example, we use the \verb+widthliketwocolumns+ and \\ \verb+continuousnumberingwithcolumns+ and \verb+\pausenumbering+ to alternate parallel typesetting in columns and typesetting in a single column, getting continuous line numbering between them.
\end{abstract}
\end{english}
\begin{pairs}  
\begin{Leftside} 
\beginnumbering  
\pstart  
Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum feugiat dolor vitae urna laoreet dapibus. Ut pulvinar nisl ultrices ex viverra, ac imperdiet sem gravida. Pellentesque malesuada augue a interdum placerat. Aenean tincidunt euismod viverra. Maecenas consectetur ligula nisi, in dignissim nulla tempus et. Aenean non scelerisque odio. 
\pend  
\pausenumbering
\end{Leftside} 

\begin{Rightside}   
\beginnumbering  
\pstart  
Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum feugiat dolor vitae urna laoreet dapibus. Ut pulvinar nisl ultrices ex viverra, ac imperdiet sem gravida. Pellentesque malesuada augue a interdum placerat. Aenean tincidunt euismod viverra. Maecenas consectetur ligula nisi, in dignissim nulla tempus et. Aenean non scelerisque odio. Phasellus cursus bibendum turpis quis finibus. Aliquam porttitor arcu sit amet consectetur varius. Suspendisse dapibus, enim quis laoreet feugiat, diam ex fringilla metus, non ullamcorper eros ante sed justo. Sed eget congue enim. Morbi dictum magna sed dictum condimentum. In mattis, velit ac facilisis eleifend, nisl orci ornare nisi, at dignissim ante purus eu lacus. Pellentesque et diam odio.
\pend  
\pausenumbering
\end{Rightside}  

\end{pairs}  
\Columns

\resumenumbering
\pstart  
Lorem \edtext{ipsum}{\Afootnote{ipsam}} dolor sit amet, consectetur adipiscing elit. Vestibulum feugiat dolor vitae urna laoreet dapibus. Ut pulvinar nisl ultrices ex viverra, ac imperdiet sem gravida. Pellentesque malesuada augue a interdum placerat. Aenean tincidunt euismod viverra. Maecenas consectetur ligula nisi, in dignissim nulla tempus et. Aenean non scelerisque odio. Phasellus cursus bibendum turpis quis finibus. Aliquam porttitor arcu sit amet consectetur varius. Suspendisse dapibus, enim quis laoreet feugiat, diam ex fringilla metus, non ullamcorper eros ante sed justo. Sed eget congue enim. Morbi dictum magna sed dictum condimentum. In mattis, velit ac facilisis eleifend, nisl orci ornare nisi, at dignissim ante purus eu lacus. Pellentesque et diam odio.
\pend  
\pausenumbering

\begin{pairs}  
 

\begin{Leftside} 
\resumenumbering  
\pstart  

Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum feugiat dolor vitae urna laoreet dapibus. Ut pulvinar nisl ultrices ex viverra, ac imperdiet sem gravida. Pellentesque malesuada augue a interdum placerat. Aenean tincidunt euismod viverra. Maecenas consectetur ligula nisi, in dignissim nulla tempus et. Aenean non scelerisque odio. Phasellus cursus bibendum turpis quis finibus. Aliquam porttitor arcu sit amet consectetur varius. Suspendisse dapibus, enim quis laoreet feugiat, diam ex fringilla metus, non ullamcorper eros ante sed justo. Sed eget congue enim. Morbi dictum magna sed dictum condimentum. In mattis, velit ac facilisis eleifend, nisl orci ornare nisi, at dignissim ante purus eu lacus. Pellentesque et diam odio.
\pend  
\pausenumbering 
\end{Leftside}

\begin{Rightside}   
\resumenumbering  
\pstart  
\edtext{Lorem}{\Afootnote{Lorim}} ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum feugiat dolor vitae urna laoreet dapibus. Ut pulvinar nisl ultrices ex viverra, ac imperdiet sem gravida. Pellentesque malesuada augue a interdum placerat. 
\pend  
\pausenumbering
\end{Rightside}  
\end{pairs}  
\Columns

\resumenumbering  
\pstart  
Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum feugiat dolor vitae urna laoreet dapibus. Ut pulvinar nisl ultrices ex viverra, ac imperdiet sem gravida. Pellentesque malesuada augue a interdum placerat. Aenean tincidunt euismod viverra. Maecenas consectetur ligula nisi, in dignissim nulla tempus et. Aenean non scelerisque odio. Phasellus cursus bibendum turpis quis finibus. Aliquam porttitor arcu sit amet consectetur varius. Suspendisse dapibus, enim quis laoreet feugiat, diam ex fringilla metus, non ullamcorper eros ante sed justo. Sed eget congue enim. Morbi dictum magna sed dictum condimentum. In mattis, velit ac facilisis eleifend, nisl orci ornare nisi, at dignissim ante purus eu lacus. Pellentesque et diam odio.
\pend  
\pausenumbering

\begin{pairs}  
  

\begin{Leftside} 
\resumenumbering  
\pstart  

Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum feugiat dolor vitae urna laoreet dapibus. Ut pulvinar nisl ultrices ex viverra, ac imperdiet sem gravida. Pellentesque malesuada augue a interdum placerat. Aenean tincidunt euismod viverra. Maecenas consectetur ligula nisi, in dignissim nulla tempus et. Aenean non scelerisque odio. Phasellus cursus bibendum turpis quis finibus. Aliquam porttitor arcu sit amet consectetur varius. Suspendisse dapibus, enim quis laoreet feugiat, diam ex fringilla metus, non ullamcorper eros ante sed justo. Sed eget congue enim. Morbi dictum magna sed dictum condimentum. In mattis, velit ac facilisis eleifend, nisl orci ornare nisi, at dignissim ante purus eu lacus. Pellentesque et diam odio.
\pend  
\pausenumbering 
\end{Leftside}

\begin{Rightside}   
\resumenumbering  
\pstart  
Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum feugiat dolor vitae urna laoreet dapibus. Ut pulvinar nisl ultrices ex viverra, ac imperdiet sem gravida. Pellentesque malesuada augue a interdum placerat. Aenean tincidunt euismod viverra. Maecenas consectetur ligula nisi, in dignissim nulla tempus et. Aenean non scelerisque odio. Phasellus cursus bibendum turpis quis finibus. Aliquam porttitor arcu sit amet consectetur varius. Suspendisse dapibus, enim quis laoreet feugiat, diam ex fringilla metus, non ullamcorper eros ante sed justo. Sed eget congue enim. Morbi dictum magna sed dictum condimentum. In mattis, velit ac facilisis eleifend, nisl orci ornare nisi, at dignissim ante purus eu lacus. Pellentesque et diam odio.
\pend  
\endnumbering
\end{Rightside} 
\end{pairs}  

\Columns
\resumenumbering  
\pstart  
Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum feugiat dolor vitae urna laoreet dapibus. Ut pulvinar nisl ultrices ex viverra, ac imperdiet sem gravida. Pellentesque malesuada augue a interdum placerat. Aenean tincidunt euismod viverra. Maecenas consectetur ligula nisi, in dignissim nulla tempus et. Aenean non scelerisque odio. Phasellus cursus bibendum turpis quis finibus. Aliquam porttitor arcu sit amet consectetur varius. Suspendisse dapibus, enim quis laoreet feugiat, diam ex fringilla metus, non ullamcorper eros ante sed justo. Sed eget congue enim. Morbi dictum magna sed dictum condimentum. In mattis, velit ac facilisis eleifend, nisl orci ornare nisi, at dignissim ante purus eu lacus. Pellentesque et diam odio.
\pend  
\endnumbering

\section{New section}

\beginnumbering  
\pstart  
Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum feugiat dolor vitae urna laoreet dapibus. Ut pulvinar nisl ultrices ex viverra, ac imperdiet sem gravida. Pellentesque malesuada augue a interdum placerat. Aenean tincidunt euismod viverra. Maecenas consectetur ligula nisi, in dignissim nulla tempus et. Aenean non scelerisque odio. Phasellus cursus bibendum turpis quis finibus. Aliquam porttitor arcu sit amet consectetur varius. Suspendisse dapibus, enim quis laoreet feugiat, diam ex fringilla metus, non ullamcorper eros ante sed justo. Sed eget congue enim. Morbi dictum magna sed dictum condimentum. In mattis, velit ac facilisis eleifend, nisl orci ornare nisi, at dignissim ante purus eu lacus. Pellentesque et diam odio.
\pend  
\pausenumbering

\begin{pairs}
\begin{Leftside} 
\resumenumbering  
\pstart  
Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum feugiat dolor vitae urna laoreet dapibus. Ut pulvinar nisl ultrices ex viverra, ac imperdiet sem gravida. Pellentesque malesuada augue a interdum placerat. Aenean tincidunt euismod viverra. Maecenas consectetur ligula nisi, in dignissim nulla tempus et. Aenean non scelerisque odio. Phasellus cursus bibendum turpis quis finibus. Aliquam porttitor arcu sit amet consectetur varius. Suspendisse dapibus, enim quis laoreet feugiat, diam ex fringilla metus, non ullamcorper eros ante sed justo. Sed eget congue enim. Morbi dictum magna sed dictum condimentum. In mattis, velit ac facilisis eleifend, nisl orci ornare nisi, at dignissim ante purus eu lacus. Pellentesque et diam odio.
\pend  
\pausenumbering 
\end{Leftside}

\begin{Rightside}   
\beginnumbering  
\pstart  
\edtext{Lorem}{\Afootnote{Lorim}} ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum feugiat dolor vitae urna laoreet dapibus. Ut pulvinar nisl ultrices ex viverra, ac imperdiet sem gravida. Pellentesque malesuada augue a interdum placerat. 
\pend  
\pausenumbering
\end{Rightside}  
\end{pairs}  
\Columns

\resumenumbering  
\pstart  
Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum feugiat dolor vitae urna laoreet dapibus. Ut pulvinar nisl ultrices ex viverra, ac imperdiet sem gravida. Pellentesque malesuada augue a interdum placerat. Aenean tincidunt euismod viverra. Maecenas consectetur ligula nisi, in dignissim nulla tempus et. Aenean non scelerisque odio. Phasellus cursus bibendum turpis quis finibus. Aliquam porttitor arcu sit amet consectetur varius. Suspendisse dapibus, enim quis laoreet feugiat, diam ex fringilla metus, non ullamcorper eros ante sed justo. Sed eget congue enim. Morbi dictum magna sed dictum condimentum. In mattis, velit ac facilisis eleifend, nisl orci ornare nisi, at dignissim ante purus eu lacus. Pellentesque et diam odio.
\pend  
\pausenumbering

\end{document}
```
What is the recommended way to achieve what I tried in the new section?

**Comment by maieul** [comment]
Sorry, I have a lot of work, and I won’t have time to look on this issue before many time

**Comment by ndlecic** [comment]
> Sorry, I have a lot of work, and I won’t have time to look on this issue before many time

There's no hurry...

**Comment by maieul** [comment]
for this problem, please try branch issue665

**Comment by ndlecic** [comment]
> for this problem, please try branch issue665

Great, this works! I'll write the promised text for the docs in the next several days. Actually, the example file and the documentation text should cover three situations:

1. Two-column text starts at the beginning of the document: we use \beginnumbering on the left and on the right side;

2. Two-column text starts after single-column test: we use \resumenumbering on the left, and \beginnumbering on the right (since no right numbering existed so far);

3. Two-column text begins right after section command, and after \endnumbering in the previous section; in this case we use \beginnumbering on the left, but \resumenumbering on the right (which is a bit counter-intuitive).

**Comment by maieul** [comment]
> 
> Two-column text begins right after section command, and after
> \endnumbering in the previous section; in this case we use
> \beginnumbering on the left, but \resumenumbering on the right (which
> is a bit counter-intuitive).
> 
> 

No, we must use beginnumbering on the right too, as we have not yet 
started a numbering.

**Comment by maieul** [comment]
hi @ndlecic when do think to have time for these MWE? And could you provide a MWE for your example 3 ?

**Comment by maieul** [comment]
I have deleted branch issue665 and merged on branch for next release 2.24.0-2.20.0. 

b3a3a810 add some remarks for use, but I you want to modify the MWE, you are welcome for a pull request on branch 2.24.0-2.20.0. For your problem 3., please open a new issue.
